### DISCUSSION TITLE
How to get flx sprites data

### DISCUSSION THREAD

**Discussion by CerBor** [HaxeFlixel/flixel#2499] [Q&A]
So, I have ```FlxSpriteGroup``` with different sprites, they also have different x positions and width. How can I get their ```BitmapData``` (with their position offset)?

**Comment by Geokureli** [comment]
"bitmapData with position offset"? Can you take a step back and tell me what you're trying to accomplish with this? You're wording suggests that you may be sidestepping the actual underlying problem.

`FlxSprites` are essentially "positioned images", and there's only very rare cases in which I would recommend looking under the hood for it's `bitmapData`

**Reply by CerBor** [reply]
Hello, I am trying to creating a text which is made from ```FlxSprite``` *(I use sparrow assets (png and xml) to split image into small letters)*. Then, I made code, which allows user to type text, using this "font", and I'm trying to do a export function. It is going to export those text (made from ```FlxSprite```) into png file.
If you still have questions, here is [my source code](https://github.com/CerBor/My-test-project).


Also, sorry for long replying.
